src/debug.c

```diff
@@ -758,7 +761,7 @@ NULL
         } else if (!strcasecmp(name,"verbatim")) {
             addReplyVerbatim(c,"This is a verbatim\nstring",25,"txt");
         } else {
-            addReplyError(c,"Wrong protocol type name. Please use one of the following: string|integer|double|bignum|null|array|set|map|attrib|push|verbatim|true|false");
+            addReplySubcommandSyntaxError(c);
```

Comment:
this one i really wanna change (merge your change), i can easily see how this one gets out of context when we add more options.
we also have the one in HELP, so i'm in favor of having just two that can get out of context.

maybe the other alternative would be better:
leave some vague `PROTOCOL <type>` message in the HELP (without specifying the list).
and have the full list here (since this list is closer to the other changes, there's less chance it'll be forgotten when one is added).

Reply:
Your way is better. But right now...
I can't think of anything better `vague PROTOCOL <type> message` ...

So i revert it right now.  
Next time when add more options, we can take care of it